packages/dtslint/src/index.ts

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 
+import { getMangledNameForScopedPackage } from "@definitelytyped/definitions-parser";
```

Comment:
Hm, I thought we had something like this in the utils package.

Reply:
Thank you for the hint. Updated.